File: python_programming/Practices/old_enough.py
#CC 7th old enough 

age = int(input("What is your age?\n"))

if age >= 18:
    print("You are old enought to vote") 
elif age >= 16 and not age > 18:
    print("You are old enough to drive") 
elif age >= 15 and not age > 16:
    print("You can get your learners permit")
elif age >=4 and not age > 15:
    print("You are old enough to go to school")
else:
    print("You are too young for anything")

File: c_programming/practices/old_enough.c
//CC 7th old enough 
#include <stdio.h>

int main(void){

    int age;
    printf("What is your age:\n");
    scanf("%d", &age); 

    if(age >=65){
        printf("You are a senior citizen, you can do everything but you might want to not drive or vote\n");
    }else if(age >=18 && age<= 65){
        printf("You are old enough to vote\n");
    }else if(age >=16 && age<= 18){
        printf("You are old enough to drive\n");
    }else if(age<= 16 && age >=15){
        printf("You are old enough to get a learners permit, wish that was me\n");
    }else{
        printf("You are old enough to go to school\n");
    }

    return 0;
}